README.md

```diff
@@ -4,7 +4,45 @@
 
 ## Contents
 
-This branch is not used for public. Please use one of the other branches available for submitting add-ons.
+This branch contains a certain category of add-ons from which our back-end script creates .zip files which are made available to each Kodi client.
+* Language add-ons
+* Image resources
+* Controller profiles
+
+## How to submit your add-on and subsequent updates ##
+
+Your add-on must follow our strict repository rules to be considered for inclusion. Please consult the [Add-on rules] (http://kodi.wiki/view/Add-on_Rules) wiki page for further details. Please review these rules carefull before submitting your add-on. Should you have any questions regarding them please start a forum thread in one of the following locations
+* [Python add-ons] (http://forum.kodi.tv/forumdisplay.php?fid=26)
+* [Skins] (http://forum.kodi.tv/forumdisplay.php?fid=12)
+
+After you have read the repository guidelines and made sure your addon is compliant with them, you may begin the submission process. By forking this repository and creating a pull-request to the correct repository branch you ask permissing to included you add-on to the official Kodi repository. Subsequent updates can be done in a similar way by updating the code and creating a new pull-request again. Make sure that your local git clone is always rebased before send a pull-request.
```

Comment:
grammar: "included you add-on to" -> "include your add-on in".

grammar: "send" -> "sending"